src/libraries/System.Text.Json/tests/System.Text.Json.Tests/System.Text.Json.Tests.csproj

```diff
@@ -269,6 +269,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\System.Text.Json.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.IO.Pipelines\src\System.IO.Pipelines.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Linq.AsyncEnumerable\src\System.Linq.AsyncEnumerable.csproj" />
```

Comment:
I think this ProjectReference is only necessary for the .NET Framework build as for NetCoreAppCurrent, the library is inbox and provided by the shared framework.

Reply:
When I do that I get this error:
```
D:\repos\runtime\src\libraries\System.Linq.AsyncEnumerable\src\System.Linq.AsyncEnumerable.csproj : error NU1101: Unable to find package System.Linq.AsyncEnumerable. No packages exist with thi
s id in source(s): C:\Program Files\dotnet\library-packs, dotnet-eng, dotnet-libraries, dotnet-libraries-transport, dotnet-public, dotnet-tools, dotnet10, dotnet10-transport [D:\repos\runtime\
src\libraries\System.Text.Json\tests\System.Text.Json.Tests\System.Text.Json.Tests.csproj]
```
Is there something else I'm missing?